README.md

```diff
@@ -15,70 +15,77 @@
 [s6]: https://tokei.rs/b1/github/amethyst/laminar?category=code
 [s7]: https://codecov.io/gh/amethyst/laminar/branch/master/graphs/badge.svg
 
-A UDP-based protocol that provides partial reliability. Coming soon!
+This library implements some TCP-like features on top of an UDP-socket. 
+I will provide a lightweight message-based interface with certain guarantees like reliability, fragmentation congestion monitoring.
 
-## Note
+Laminar was designed to be used in the [Amethyst][amethyst] game engine and is mostly based on articles from [gaffer on games](https://gafferongames.com/).
 
-This library is not yet stable. It is experimental and things may change frequently.
+[amethyst]: https://github.com/amethyst/amethyst
 
 ## Table of contents:
-- [Useful links](https://github.com/amethyst/laminar#useful-links)
-- [Features](https://github.com/amethyst/laminar#features)
-- [Examples](https://github.com/amethyst/laminar#examples)
-    - [Udp](https://github.com/amethyst/laminar#udp)
-- [Notice](https://github.com/amethyst/laminar#notice)
-- [Contributing](https://github.com/amethyst/laminar#contributing)
-- [Authors](https://github.com/amethyst/laminar/#authors)
+- [Useful links](#useful-links)
+- [Features](#features)
+- [Getting Started](#getting-stated)
+- [Examples](#examples)
+- [Notice](#notice)
+- [Contributing](#contribution)
+- [Authors](#authors)
 - [License](#license)
 
+## Features
+These are the features this crate provides:
+
+- UDP-based protocol
+- Automatic Fragmentation
+- RTT estimation
+- Connection tracking.
```

Comment:
We should keep this list consistent and remove the periods at the end of the items.